.github/workflows/npm-brew.yaml

```diff
@@ -19,10 +19,9 @@ jobs:
       - name: Publish npm package and tag with "latest"
         run: yarn publish:npm
         env:
-          ENVIRONMENT: "production"
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-          NPM_TAG: "latest"
+          NPM_ENVIRONMENT: "production"
```

Comment:
✍🏼 Shellcheck didn't like me using `ENVIRONMENT`. Probably safer to make it more specific in case GH or something else uses `ENVIRONMENT`.